packages/mostly-equal/src/safe-print.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+import type { Path } from './types';
+
+export const spaces = (indent: number) => {
+  const arr = new Array(indent);
+  arr.fill('    ');
+  return arr.join('');
+};
+
+export const printPath = (p: Path) => {
+  return `actual${p.map((item) => (typeof item === 'number' ? `[${item.toString()}]` : `.${item}`)).join('')}`;
+};
+
+export const safePrint = (
```

Comment:
remove this function. rename `safePrintRecurse` to safePrint`, and give it default params there. the extra function does nothing other than create the default values

Reply:
done